product_docs/docs/pgd/5.6/security/pgd-predefined-roles.mdx

```diff
@@ -150,6 +150,7 @@ This role is designed for applications that require access to PGD features, obje
 -   [`bdr.trigger_get_xid`](/pgd/latest/reference/streamtriggers/rowfunctions#bdrtrigger_get_xid)
 -   [`bdr.wait_for_camo_partner_queue`](/pgd/latest/reference/functions#bdrwait_for_camo_partner_queue)
 -   [`bdr.wait_slot_confirm_lsn`](/pgd/latest/reference/functions#bdrwait_slot_confirm_lsn)
+-   [`bdr.wait_node_confirm_lsn`](/pgd/latest/reference/functions#bdrwait_node_confirm_lsn)
```

Comment:
Please move this to bdr_superuser as those are the permissions needed for wait_node_confirm_lsn for 5.6.1